stroeer/core/v1/shared.proto

```diff
@@ -6,31 +6,34 @@ option go_package = "github.com/stroeer/tapir/go/stroeer/core/v1;core";
 option java_multiple_files = true;
 option java_package = "de.stroeer.core.v1";
 
+// A Reference represents a link to another entity - for example an `Article`,
```

Comment:
Maybe we should add the fact, that a reference can be a whole tree structure now in this comment as well